pkg/api/defaults.go

```diff
@@ -99,7 +99,7 @@ func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool) {
 		// we translate deprecated NetworkPolicy usage to the NetworkConfig equivalent
 		// and set a default network policy enforcement configuration
 		switch o.KubernetesConfig.NetworkPolicy {
-		case NetworkPluginAzure:
+		case NetworkPolicyAzure:
```

Comment:
@CecileRobertMichon I made this sort of superficial change because (1) it's more expressive to use the `NetworkPolicy` const here and (2) hopefully it answers your question below.

Reply:
Sorry, your question above ;)